js/layout-javascript/agenda-code.js

```diff
@@ -587,17 +591,17 @@ DynamicList.prototype.prepareData = function(records) {
           record.data['modified_' + field.column] = record.data['modified_' + field.column];
         }
 
-        columns.push('data[modified_' + field.column + ']');
```

Comment:
For a 3-field pre-sorted instance, this grew to about ~6700 instances of the same 3 column names, which later resulted in exponentially more loops in the `orderBy`